pages.fr/common/git-diff.md

```diff
@@ -15,7 +15,7 @@
 
 `git diff --staged`
 
-- Afficher les changements de tous les commits à partir d'une date / heure donnée (expression de dates, ex : "1 week 2 days" ou une date ISO) :
+- Afficher les changements de tous les commits à partir d'une date / heure donnée (expression de dates, ex : "1 week 2 days" pour 1 semaine et 2 jours ou une date ISO) :
```

Comment:
What exactly did you change here? The former version looks more like the literal translation of the english version.

Reply:
You're right. I just translated the example given in English "1 week 2 days" in French "1 semaine et 2 jours" for those who do not understand English.